skill-prompt-generator/README_v2.0.md

```
@@ -0,0 +1,196 @@
+# 扩展数据库，添加变量表
+sqlite3 extracted_results/elements.db < core/schema_migration_v1.sql
+```
+
+### 2. 依赖检查
+
+```bash
+python3 -c "import yaml; print('✅ PyYAML installed')"
+```
+
+如果未安装：
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)